docs/dev_guide/contents/extending_fido.rst

```diff
@@ -0,0 +1,187 @@
+.. _extending_fido:
+
+***************************************
+Extending Fido with New Sources of Data
+***************************************
+
+The `~sunpy.net.fido_factory.UnifiedDownloaderFactory` (``Fido``) object is extensible with new clients, which can interface with web services and download new data or metadata.
+There are two ways of defining a new client, depending on the complexity of your web service.
+A "simple" client inherits from `~sunpy.net.dataretriever.client.GenericClient` which provides helper methods for downloading from a list of URLs.
+If your web service provides a list of HTTP or FTP urls that can easily be obtained from a search, this is probably the route to go.
+If your web service requires you to do complex parsing of the search, or needs to construct specific objects to interface with the web service, or you need control over the download implementation (i.e. does not just return a list of URLs) then you probably want to write a "full" client.
+
+Writing a new "simple" client
```

Comment:
I think there should be a sentence here somewhere saying that this is done through using scraper

Reply:
I renamed this whole section to "scraper client" as it seemed more descriptive.